src/main/webapp/resources/js/components/samples/components/SampleFileContenate.tsx

```diff
@@ -19,21 +19,32 @@ import {
   resetConcatenateSelected,
 } from "../sampleFilesSlice";
 import { IconArrowLeft, IconArrowRight, IconFile } from "../../icons/Icons";
+import {
+  SampleSequencingObject,
+  SequencingFile,
+  SequencingObject,
+} from "../../../apis/samples/samples";
```

Comment:
Duplicated imports here as well

Reply:
Cleaned up in 1398b80e9ff839bad02c6c0dfd63fabd2e3bf8a5